ui/Screen/ProjectCreation.svelte

```diff
@@ -198,9 +195,9 @@
     beginValidation = true;
 
     try {
-      localBranches = await getLocalBranches(path);
+      localState = await getLocalState(path);
```

Comment:
The branch selector should not contain any remote branches. See: https://github.com/radicle-dev/radicle-upstream/issues/434.